intro-HPC/brussel/macros.tex

```diff
@@ -4,6 +4,8 @@
 \macro{\association}{Brussels University Association (BUA)}
 % HPC definitions
 \macro{\hpc}{VUB-HPC}
+\macro{\hpcInfra}{VUB-HPC}
+\macro{\hpcTeam}{VUB-HPC}
```

Comment:
because I just copy pasted the line and changed the name of the macro,
I expect people will actually want to change it, or will want to change the definition of `\hpc`, so this allows them to change `\hpc` without having to also change the infra and team macro.